lib/goodcity/organisation_populator.rb

```diff
@@ -22,7 +22,7 @@ def initialize
     end
 
 
-    def populate_organisation_db
+    def run
```

Comment:
@trojanh I think the class name made it clear what the job is so no need to repeat in the name